components/header/navbar/index.tsx

```diff
@@ -92,7 +95,13 @@ const Navbar = ({
             </>
           )}
         </div>
-        <LoginButton className={position === 'side' ? 'mt-6' : ''} />
+
+        <Row className={position === 'side' ? 'mt-6' : ''}>
```

Comment:
Would be good to have more space between log out and search buttons on mobile, either just with larger margins or with justify-around (similarly to how its done for täffä & lunch above)
```suggestion
        <Row
          className={classNames(
            'justify-around',
            position === 'side' ? 'mt-6' : ''
          )}
        >
```